src/Microsoft.AspNetCore.Hosting/WebHostBuilder.cs

```diff
@@ -228,17 +219,17 @@ private IServiceCollection BuildHostingServices()
             return services;
         }
 
-        private string ResolveApplicationBasePath(string applicationBasePath, string basePath)
+        private string ResolveContentRootPath(string contentRootPath, string basePath)
         {
-            if (string.IsNullOrEmpty(applicationBasePath))
+            if (string.IsNullOrEmpty(contentRootPath))
```

Comment:
This is how it worked before as well. We need the env variable because httpPlatformHandler/AspNetCoreModule start the application in wwwroot making the wwwroot the baseApplicationPath which prevents from finding views. The env variable changes the path to be the folder where the application dlls live which contain sthe Views folder.